### PR TITLE
CMakeLists file updated

### PR DESCRIPTION
**Pull request by SatjaSivcev**
- Version requirement removed.
- Copying dependencies made universal.
- This file can now be used with Visual Studio and QT creator.

### CODE REVIEW

**Comment by nedrebo** [comment]
You should add relevant reviewers to your PRs to draw attention. Then it will show up in the reviewers dashboards.

I would have added me, but especially @apartridge since he wrote the initial code and he did not participate in writing this patch. Me reviewing this would be less valuable since I wrote most of this patch.

**Comment by nedrebo** [comment]
The other top level CMakeLists.txt also needs updating.

As discussed before, I think you should consider having only one top level CMakeLists.txt. That would reduce code duplication.

File: ZividPCL/CMakeLists.txt [hunk 1]
```diff
@@ -22,14 +32,7 @@ foreach(SAMPLE_NAME ${SAMPLE_VIS_NAMES})
 		Zivid::Vis3D
 		${PCL_LIBRARIES}
 	)
+	
+	add_dependencies(${SAMPLE_NAMES} SampleDependencies)
```
Comment (knatten) [inline ZividPCL/CMakeLists.txt]: This should say `SAMPLE_NAME` instead of `SAMPLE_NAMES`.

File: Zivid/CMakeLists.txt [hunk 2]
```diff
@@ -17,13 +27,7 @@ foreach(SAMPLE_NAME ${SAMPLE_NAMES})
 		Zivid::Vis3D
 	)
 
+    add_dependencies(${SAMPLE_NAMES} SampleDependencies)
```
Comment (knatten) [inline Zivid/CMakeLists.txt]: This should say `SAMPLE_NAME` instead of `SAMPLE_NAMES`.

**Comment by knatten** [comment]
Talked to @nedrebo now, let's merge this as is, and then solve the thing with a common CMakeLists as a separate issue. Created #7 for that.

File: Zivid/CMakeLists.txt (hunk 2 above)
Comment (knatten) [reply Zivid/CMakeLists.txt]: Remember to do this on the downsample branch as well btw.

**Pull request merged**